native/chat/chat-thread-background.react.js

```diff
@@ -34,10 +36,38 @@ export default function BackgroundChatThreadList(
     props.navigation.setOptions({ title });
   }, [props.navigation, unreadBackgroundThreadsNumber]);
 
+  const textStyle =
```

Comment:
Instead of deciding on the colors in the component, let's try to use the colors defined in the themes in `native/themes/colors.js`. This way we can use the same colors everywhere, and if we want to change any colors, we only need to edit one place.

In this case, since it looks like `ChatThreadList` has `backgroundColor: 'listBackground'`, we can use `color: 'listBackgroundLabel'` for our text. This is similar to what you've selected, only it uses `#C7C7CC` instead of `white`. (Let me know if it makes the text too hard to see, I haven't actually checked what it looks like. An alternative would be `listForegroundLabel`, which is exactly the same colors you're using here.)

To see how we can use these colors in a Hook, check out `native/chat/new-messages-pill.react.js`. You basically define the style sheet pretending that `listBackgroundLabel` is a color, and then you pass it to `useStyles`, which will decide if it's a light or dark background.

Note that you don't need to call `StyleSheet.create` in this case. `useStyles` will call it for you, and I think in `DEV` mode `StyleSheet.create` can error if you give it a color it doesn't understand.